extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/util/MultiStageQueryContext.java

```diff
@@ -60,6 +61,9 @@
   public static final String CTX_ENABLE_DURABLE_SHUFFLE_STORAGE = "durableShuffleStorage";
   private static final boolean DEFAULT_ENABLE_DURABLE_SHUFFLE_STORAGE = false;
 
+  public static final String CTX_ENABLE_DURABLE_TASK_INTERMEDIATE_STORAGE = "durableTaskIntermediateStorage";
```

Comment:
I think that we should somehow try to merge this config property with `durableShuffleStorage` to make it easier for the users to configure. Adding another property means that the user can now have up to 4 configurations of `durableShuffleStorage` and `durableTaskIntermediateStorage` each with different semantics. WDYT?

Reply:
I had initially added the separate feature flag because durable storage reads were coming to be very slow - which would lead to slow UTs and also slow unintended user queries. Now that #13373 is merged, I have removed the extra flag. Also made the super sorter test run with both file based and durable storage based output channels